scripts/sts/create-rhoam-policy.sh

```diff
@@ -147,16 +146,16 @@ cat <<EOM >"$MINIMAL_POLICY_NAME.json"
                 "ec2:DeleteVpc",
                 "ec2:DeleteVpcPeeringConnection",
                 "elasticache:BatchApplyUpdateAction",
-                "elasticache:CreateCacheSubnetGroup",
                 "elasticache:CreateSnapshot",
                 "elasticache:DeleteCacheSubnetGroup",
                 "elasticache:DeleteSnapshot",
-                "elasticache:DescribeSnapshots",
-                "elasticache:DescribeUpdateActions",
                 "elasticache:ModifyCacheSubnetGroup",
+                "elasticache:ModifyReplicationGroup",
                 "rds:DeleteDBInstance",
                 "rds:DeleteDBSnapshot",
-                "rds:DeleteDBSubnetGroup"
+                "rds:DeleteDBSubnetGroup",
+                "rds:ModifyDBInstance",
+                "rds:RemoveTagsFromResource"
```

Comment:
@cecobask do the removals/additions here impact cases where IAM is the authentication method?

Reply:
actually nvm. I see this is the STS role file not the IAM Credentials Request. 👍